packages/connect/src/data/config.ts

```diff
@@ -197,11 +197,11 @@ export const config = {
                 'setBusy',
                 'unlockPath',
             ],
-            min: ['1.12.1', '2.5.3'],
+            min: { T1B1: '1.12.1', T2T1: '2.5.3' },
         },
         {
             methods: ['showDeviceTutorial'],
-            min: ['0', '2.6.1'],
+            min: { T1B1: '0', T2T1: '0', T2B1: '2.6.1' },
```

Comment:
ok, t2b1 only for specific cases, this config acts as a blacklist, so we dont need to specify t2b1 everywhere